examples/realtime/plot_compute_rt_average.py

```diff
@@ -55,6 +55,7 @@
         evoked = ev
     else:
         evoked += ev
+    evoked.pick_types(meg=True, eog=False)  # leave out the eog channel
```

Comment:
Yes because only one axes is provided to the plotting function. If both types are plotted, a second axes should be provided.